modules/vf-graphql-holochain/queries/agreement.ts

```diff
@@ -5,19 +5,20 @@
  * @since:   2020-06-16
  */
 
-import { DNAIdMappings } from '../types'
+import { DNAIdMappings, ReadParams } from '../types'
 import { mapZomeFn } from '../connection'
 
 import {
-  Agreement,
+  Agreement, AgreementResponse,
 } from '@valueflows/vf-graphql'
 
 export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
-  const readRecord = mapZomeFn(dnaConfig, conductorUri, 'agreement', 'agreement', 'get_agreement')
+  const readRecord = mapZomeFn<ReadParams, AgreementResponse>(dnaConfig, conductorUri, 'agreement', 'agreement', 'get_agreement')
 
   return {
-    agreement: async (root, { id }): Promise<Agreement> => {
-      return (await readRecord({ address: id })).agreement
+    // why does agreement query not return AgreementResponse like other queries?
```

Comment:
I'm not sure I follow your comment/question here. It appears to me to follow in the same pattern as the others

Reply:
ah, looks unintended

Reply:
as follow up, I logged this: https://lab.allmende.io/valueflows/vf-schemas/vf-graphql/-/issues/99